requirements.txt

```diff
@@ -1,3 +1,3 @@
 cython
 virtualenv
-git+https://github.com/endlessm/python-for-android@v2022.09.04-endless8#egg=python-for-android
+git+https://github.com/endlessm/python-for-android@51f2107a62248feb30f3d21338674fbfb6176184#egg=python-for-android
```

Comment:
2nd commit is great for testing the p4a changes along with these ones in our CI build! I will approve without merging so you can update this line after tagging in the p4a repo.

Reply:
Okay, I changed this to refer to `v2022.09.04-endless9` in the newest push.